int number_of_tries
string camera_name
---
int8[] results